css/hl.css

```diff
@@ -4608,6 +4617,47 @@ pre .hs-keyglyph {
     -webkit-flex: 0 1 auto;
         -ms-flex: 0 1 auto;
             flex: 0 1 auto; }
-    .row-flex > div[class*='col-'] .thumbnail {
+  .row-flex > div[class*='col-'] .thumbnail {
       width: 100%; }
 
+  .glider,.glider-contain{margin:0 auto;position:relative;}
```

Comment:
Would it be worth getting a CSS formatter to make the CSS a bit more uniform and readable? I haven't used one before, so I can't recommend any particular tool.

Reply:
Yeah, I agree we probably should look into that if we can find a good one.